pkg/azclient/policy/etag/etag.go

```diff
@@ -45,7 +43,9 @@ func AppendEtag(req *policy.Request) (*http.Response, error) {
 		if etag.ETag != "" {
 			req.Raw().Header.Set("If-Match", string(etag.ETag))
 		}
-		req.SetBody(streaming.NopCloser(strings.NewReader(string(body))), "application/json")
+		if err = req.RewindBody(); err != nil {
```

Comment:
Can we unit test this and other similar changes?

Reply:
removed the change becuase more issues are found